src/components/TidligstMuligUttaksalder/TidligstMuligUttaksalder.tsx

```diff
@@ -28,6 +29,15 @@ export const TidligstMuligUttaksalder: React.FC<Props> = memo(
         <Ingress className={styles.ingress}>
           Jo lenger du venter, desto mer får du i året.
         </Ingress>
+        <div className={styles.info}>
+          <InformationSquareFillIcon
```

Comment:
Ser ut som denne må ha alt-text for å oppfylle a11y-kravene i smoketesten

Reply:
Jeg tror at det er mer riktig ift WCAG at det skal ha aria-hidden i og med at det ikonet er dekorativt. Jeg lager ny commit.